src/main/java/org/opensearch/security/tools/SecurityAdmin.java

```diff
@@ -832,15 +831,13 @@ private static boolean uploadFile(final RestHighLevelClient restHighLevelClient,
 	private static boolean uploadFile(final RestHighLevelClient restHighLevelClient, final String filepath, final String index, final String _id, final boolean legacy, boolean resolveEnvVars,
         final boolean populateEmptyIfMissing) {
 
-        String type = "_doc";
         String id = _id;
                 
         if(legacy) {
-            type = "security";
             id = _id;
 
             try {
-                ConfigHelper.fromYamlFile(filepath, CType.fromString(_id), 1, 0, 0);
+                ConfigHelper.fromYamlFile(filepath, CType.fromString(_id), 2, 0, 0);
```

Comment:
This was for migration Config 6 -> Config 7. But OpenSearch doesn't need to support this migration. Removing the 2 test classes for the same reason.

Reply:
Can we include documentation in the code as to how this value is determine or aligned to.  This could also be done with a local variable eg. `final int latestSupportedMajorVersion = 2;`

Reply:
That's a good idea! Can we do that in a separate PR? Let's unblock 2.0.0 build first.